migrator/migrations/m_186_to_m_187_add_blob_search/migration.go

```diff
@@ -0,0 +1,85 @@
+package m186tom187
```

Comment:
This could be extracted as a generic tool or a code generator function. We can do it later.

Reply:
you should probably write a ticket for that now while it is fresh.  

Also @rhybrillou wrote some make tools to does some of the basic generation.  Did you look at any of that to template this out?